app/assets/javascripts/semantic-ui.js

```diff
@@ -0,0 +1,19 @@
+//= require semantic-ui/modules/accordion.js 
```

Comment:
hm, I'm pretty sure that
```
//= require semantic-ui/modules/
```
would work perfectly without all this 19-lines specifications